node/core/candidate-validation/Cargo.toml

```diff
@@ -5,11 +5,11 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
-futures = "0.3.5"
+futures = "0.3.8"
 log = "0.4.11"
 
 sp-core = { package = "sp-core", git = "https://github.com/paritytech/substrate", branch = "master" }
-parity-scale-codec = { version = "1.3.0", default-features = false, features = ["bit-vec", "derive"] }
+parity-scale-codec = { version = "1.3.5", default-features = false, features = ["bit-vec", "derive"], package = "parity-scale-codec" }
```

Comment:
If I [understand `package` correctly](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#renaming-dependencies-in-cargotoml), one can use it to rename dependencies. Why is it needed here, given that it does not rename the import variable, but instead just repeats the name?

Reply:
This was a bug in `cargo-edit` tool that I used to automatically bump all the dependencies. Reverted this change manually for `futures` in 3d16db4, thanks for catching this for `parity-scale-codec`, will fix now.